tests/AttributeEventsTest.php

```diff
@@ -1,34 +1,51 @@
 <?php
 
-namespace Kleemans\Test;
+namespace Kleemans\Tests\Unit;
 
 use Illuminate\Database\Capsule\Manager as DB;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Events\Dispatcher;
 use Illuminate\Support\Testing\Fakes\EventFake;
+use Kleemans\Tests\Fake\Events\InvoiceDownloaded;
+use Kleemans\Tests\Fake\Events\OrderCanceled;
+use Kleemans\Tests\Fake\Events\OrderDeleted;
+use Kleemans\Tests\Fake\Events\OrderMadeFree;
+use Kleemans\Tests\Fake\Events\OrderMetaUpdated;
+use Kleemans\Tests\Fake\Events\OrderNoteUpdated;
+use Kleemans\Tests\Fake\Events\OrderPaid;
+use Kleemans\Tests\Fake\Events\OrderPaidHandlingFee;
+use Kleemans\Tests\Fake\Events\OrderPaidWithCash;
+use Kleemans\Tests\Fake\Events\OrderPlaced;
+use Kleemans\Tests\Fake\Events\OrderReturned;
+use Kleemans\Tests\Fake\Events\OrderShipped;
+use Kleemans\Tests\Fake\Events\OrderShippingCountryChanged;
+use Kleemans\Tests\Fake\Events\OrderTaxCleared;
+use Kleemans\Tests\Fake\Events\OrderUpdated;
+use Kleemans\Tests\Fake\Events\PaypalPaymentDenied;
+use Kleemans\Tests\Fake\Order;
```

Comment:
Imo it's more explicit to use the namespaces inline. Could you revert this?

Reply:
Okay will revert it .. At max tomorrow